packages/notifications/push/amplify_push_notifications_pinpoint/lib/src/pinpoint_provider.dart

```diff
@@ -213,15 +226,15 @@ class PinpointProvider implements ServiceProviderClient {
     return EventInfo(source, analyticsProperties);
   }
 
-  // ChannelType? _getChannelType() {
-  //   if (Platform.isAndroid) {
-  //     return ChannelType.gcm;
-  //   } else if (Platform.isIOS) {
-  //     if (zDebugMode) {
-  //       return ChannelType.apnsSandbox;
-  //     }
-  //     return ChannelType.apns;
-  //   }
-  //   return null;
-  // }
+  ChannelType? _getChannelType() {
+    if (Platform.isAndroid) {
+      return ChannelType.gcm;
+    } else if (Platform.isIOS) {
+      if (zDebugMode) {
+        return ChannelType.apnsSandbox;
+      }
+      return ChannelType.apns;
+    }
+    return null;
```

Comment:
When would the platform not be iOS or Android? I would make this return `ChannelType`